<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileListForm</class>
 <widget class="QWidget" name="FileListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_Up">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/ffftpgui.qrc">
         <normaloff>:/folderup.png</normaloff>:/folderup.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Open">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/ffftpgui.qrc">
         <normaloff>:/folderopen.png</normaloff>:/folderopen.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_DirHist">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView_DirView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::DoubleClicked</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView_DirView">
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::ResizeMode::Adjust</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/ffftpgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton_Up</sender>
   <signal>clicked()</signal>
   <receiver>FileListForm</receiver>
   <slot>onClick_toolButton_Up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>12</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_Open</sender>
   <signal>clicked()</signal>
   <receiver>FileListForm</receiver>
   <slot>onClick_toolButton_Open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_DirHist</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FileListForm</receiver>
   <slot>onCurrentIndexChange_comboBox_DirHist(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_DirHist</sender>
   <signal>textActivated(QString)</signal>
   <receiver>FileListForm</receiver>
   <slot>onTextActivate_comboBox_DirHist(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView_DirView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>FileListForm</receiver>
   <slot>onDoubleClick_treeView_DirView(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClick_toolButton_Up()</slot>
  <slot>onClick_toolButton_Open()</slot>
  <slot>onCurrentIndexChange_comboBox_DirHist(int)</slot>
  <slot>onTextActivate_comboBox_DirHist(QString)</slot>
  <slot>onDoubleClick_treeView_DirView(QModelIndex)</slot>
 </slots>
</ui>
